src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -826,6 +826,7 @@ SymbolKind.Local or
 
                 // If this is a parameter symbol for a partial method definition, be sure we visited 
                 // the implementation part's body.
+                // https://github.com/dotnet/roslyn/issues/73772: also do this for properties
```

Comment:
This means if the renaming symbol  is a parameter of a partial method, make sure rename engine also checks the method for conflict.
I don't think this would apply to partial property. 
The only similar case would be if the property is  a indexer, but in that case the propertie would be `this[int bar]`. So still don't need to check if `this` would case conflict

Reply:
```suggestion
```